engine/gamesys/src/dmsdk/gamesys/script.h

```diff
@@ -153,31 +155,14 @@ namespace dmScript
             LuaBufferOwnership  m_Owner;
         };
 
-        LuaHBuffer() {}
-
-        LuaHBuffer(dmBuffer::HBuffer buffer, LuaBufferOwnership ownership)
-        : m_Buffer(buffer)
-        , m_Owner(ownership)
-        {
-        }
+        uint16_t m_BufferResVersion;
+        dmhash_t m_BufferResPathHash;
 
-        LuaHBuffer(void* buffer_resource)
-        : m_BufferRes(buffer_resource)
-        , m_Owner(OWNER_RES)
-        {
-        }
-
-        LuaHBuffer(dmBuffer::HBuffer buffer, bool use_lua_gc)
-        : m_Buffer(buffer)
-        , m_UseLuaGC(use_lua_gc)
-        {
-            static int first = 1;
-            if (first)
-            {
-                first = 0;
-                dmLogWarning("The constructor is deprecated: dmScript::LuaHBuffer wrapper = { HBuffer, bool };");
-            }
-        }
+        LuaHBuffer();
+        LuaHBuffer(dmBuffer::HBuffer buffer, LuaBufferOwnership ownership);
+        LuaHBuffer(dmResource::HFactory factory, void* buffer_resource);
```

Comment:
Breaking change? I don't think may are using this tbh.

Reply:
And even if it is a breaking change, it's in the C++ SDK which is not nearly as used.